### ISSUE TITLE
Scaffolding doesn't work on .NET Core

### ISSUE THREAD

**Issue by bricelam** [efcore/EFCore.ConfigurationManager#2]
This is currently blocked by https://github.com/dotnet/runtime/issues/931. This is the error you see:

> A named connection string was used, but the name 'MyConnection' was not found in the application's configuration.

### Workaround

You can work around this by renaming the .config file in the output directory (aka bin\Debug) to `ef.dll.config`.

**Comment by bricelam** [comment]
Duplicate of https://github.com/dotnet/efcore/issues/22205
